src/utils/vars.css

```diff
@@ -0,0 +1,10 @@
+:root {
+  --primary-brand-color: #24cca7;
+  --main-brand-color: #4a56e2;
```

Comment:
czemu primary i main? semantycznie to jakoś rozdzielić trzeba, może accent lub coś innego bardziej pasującego